lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -2,10 +2,20 @@ module Fastlane
   module Actions
     class IosLintLocalizationsAction < Action
       def self.run(params)
-        violations = {}
+        violations = Hash.new([])
 
         loop do
-          violations = self.run_linter(params)
+          # If we did `violations = self.run...` we'd lose the default value for missing key being `[]` that we set above with `Hash.new`.
+          # We want that default value so that we can use `+=` when adding the duplicate keys violations below.
+          violations = violations.merge(self.run_linter(params))
+
+          if params[:check_duplicate_keys]
+            find_duplicated_keys(params).each do |language, duplicates|
+              violations[language] += duplicates
+            end
+          end
```

Comment:
I went for making the action call the helper to find the duplicate keys, rather than moving the logic in `L10nLinterHelper` because it seemed good for each helper to do their own thing.

But, I don't feel strongly about it, and I do like a Fastlane action that has little logic in it. I could be easily persuaded to move the logic in `L10nLinterHelper`, by adding a `check_duplicate_keys` parameter to the `run` action.

Reply:
Keeping this `find_duplicated_keys` method in the action is fine by me, no need to move it into the helper imho, especially since the method is short enough (and even more so if you adopt [my nitpick suggestion to shorten it even further](https://github.com/wordpress-mobile/release-toolkit/pull/360/files#r861164457) 😄 )